src/protocols/ethereum/libs/EtherscanService.ts

```diff
@@ -112,6 +116,19 @@ export class EtherscanService {
     ) || [];
   }
 
+  async fetchAccountTokenTransactionByHash(
+    hash: string,
+    address: string,
+    blockNumber: string,
+  ): Promise<ITransaction> {
+    // Not the best solution, but it seems to be the only way to get token transaction details
```

Comment:
We can use ethplorer, to fetch this info.
https://sepolia-api.ethplorer.io/getTxInfo/0x03cc5ebf3001fc8dce167ffcbc10896aed4c7dad10996e4d652ac50c6f4f2559?apiKey=freekey

Reply:
Ethplorer's response is missing fields like `gasPrice` and `nonce`